src/context/walletContext/walletContext.tsx

```diff
@@ -28,18 +26,12 @@ interface WalletContextType {
     walletColor: string
 }
 
-function isPeanut(wallet: interfaces.IWallet | undefined) {
+function isPeanut(wallet: interfaces.IDBWallet | undefined) {
     return wallet?.walletProviderType === interfaces.WalletProviderType.PEANUT
 }
```

Comment:
nit: function name seems quite short - I'd prefer overly long but descriptive names over too short generic ones that might cause confusion down the line